airflow/cli/commands/info_command.py

```diff
@@ -27,11 +27,11 @@
 
 import requests
 import tenacity
+from typing_extensions import Protocol
 
 from airflow import configuration
 from airflow.cli.simple_table import AirflowConsole
 from airflow.providers_manager import ProvidersManager
-from airflow.typing_compat import Protocol
```

Comment:
We have to import Prootcol directly for sphinx-autoapi to understand this correctly.

Reply:
This is done this way for compatibility across multiple python versions.

Reply:
I install `typing_extensions` on every pythton version, because `sphinx-autoapi` has problems with it. 
```
WARNING: Cannot resolve import of airflow.typing_compat.Protocol in airflow.stats
```
Hopefully we can fix this differently, but for now, I wanted to take a step forward to see if we had other critical issues. Now the documentation for the apache-airflow package builds up fine, so I think it's a pretty cool start.